### ISSUE TITLE
Show the status of the hackerspace on the widget

### ISSUE THREAD

**Issue by FriederHannenheim** [spaceapi-community/my-hackerspace#26]
I would like to see if my hackerspace is open at a glance on the widget. That way I wouldn't have to open the app every time I want to visit.

**Comment by niccokunzmann** [comment]
Hi, two ways are there: text in the settings as a checkbox and  implementing the status-icon in the api.
maybe adding text as default when there is no specific icon would be a solution? What do you think?

**Comment by FriederHannenheim** [comment]
Yeah text as default would be great. I now realized that it is already possible. This needs to be explained somewhere. There is no way a user can know that the icon shows the status of the hackerspace. It isn't even in the README
